app/screens/settings-screen/show-nostr-secret.tsx

```diff
@@ -0,0 +1,42 @@
+import useNostrProfile from "@app/hooks/use-nostr-profile"
+import ReactNativeModal from "react-native-modal"
+import Clipboard from "@react-native-clipboard/clipboard"
+import { View } from "react-native"
+import { Text } from "@rneui/themed"
+import { useState } from "react"
+
+interface ShowNostrSecretProps {
+  isActive: boolean
+  onCancel: () => void
+}
+
+export const ShowNostrSecret: React.FC<ShowNostrSecretProps> = ({
+  isActive,
+  onCancel,
+}) => {
+  const { nostrSecretKey } = useNostrProfile()
+  const [copied, setCopied] = useState(false)
+
+  const copyToClipboard = () => {
+    Clipboard.setString(nostrSecretKey)
+    setCopied(true)
+  }
+
+  return (
+    <ReactNativeModal
+      isVisible={isActive}
+      backdropColor="white"
+      backdropOpacity={0.9}
+      onBackButtonPress={onCancel}
+      onBackdropPress={onCancel}
+    >
+      <View style={{ backgroundColor: "white" }}>
+        <Text type="h2">Your nostr address is: {"\n"}</Text>
+        <Text>
+          {nostrSecretKey} {"\n"}
+        </Text>
+        <Text onPress={copyToClipboard}>{copied ? "Copied!" : "Click to copy"}</Text>
+      </View>
+    </ReactNativeModal>
+  )
+}
```

Comment:
![Screenshot_1708022416](https://github.com/lnflash/flash-mobile/assets/57404324/4107deb0-9acb-4b92-805b-fa537ec2dad6)

Can you make better design for this modal, please?

Reply:
@abhay-raizada we have been using https://bitcoin.design as a guide for many of the screens, you might find some good templates there.

Reply:
Sure, on it

Reply:
done